lib/src/barback/dartdevc/dartdevc_environment.dart

```diff
@@ -83,6 +87,15 @@ class DartDevcEnvironment {
       allFutureAssets
           .addAll(_buildAsset(module.jsId, logError: logError).values);
     }
+    // Copy all js resoureces for each of the app dirs that were discovered.
```

Comment:
At this point you might want to write a github-pr bot that adds this comment for you